src/Core/Components/SortableList/FluentSortableList.razor.cs

```diff
@@ -0,0 +1,84 @@
+using System.Diagnostics.CodeAnalysis;
+using Microsoft.AspNetCore.Components;
+using Microsoft.FluentUI.AspNetCore.Components.Utilities;
+using Microsoft.JSInterop;
+
+namespace Microsoft.FluentUI.AspNetCore.Components;
+
+public partial class FluentSortableList<TItem> : FluentComponentBase
+{
+    private const string JAVASCRIPT_FILE = "./_content/Microsoft.FluentUI.AspNetCore.Components/Components/SortableList/FluentSortableList.razor.js";
+
+    [Inject]
+    private IJSRuntime JSRuntime { get; set; } = default!;
+
+    [Parameter]
+    public RenderFragment<TItem>? SortableItemTemplate { get; set; }
+
+    [Parameter, AllowNull]
+    public List<TItem> Items { get; set; }
+
+    [Parameter]
+    public EventCallback<(int oldIndex, int newIndex)> OnUpdate { get; set; }
+
+    [Parameter]
+    public EventCallback<(int oldIndex, int newIndex)> OnRemove { get; set; }
+
+    [Parameter]
+    public string? Group { get; set; }
+
+    [Parameter]
+    public string? Pull { get; set; }
+
+    [Parameter]
+    public bool Put { get; set; } = true;
+
+    [Parameter]
+    public bool Sort { get; set; } = true;
+
+    [Parameter]
+    public string Handle { get; set; } = string.Empty;
+
+    [Parameter]
+    public string Filter { get; set; } = string.Empty;
+
+    protected string? ClassValue => new CssBuilder(Class)
+        .AddClass("fluent-sortable-list")
+        .Build();
+
+    protected string? StyleValue => new StyleBuilder(Style)
+        .Build();
+
+    private DotNetObjectReference<FluentSortableList<TItem>>? selfReference;
+
+    public FluentSortableList()
```

Comment:
Can you place the members in a clearly defined order?

See **Can you place the members in a clearly defined order?** in 
https://dvoituron.com/naming-conventions/rules/